ARM-wvd-templates/HCI/Readme.md

```diff
@@ -0,0 +1,26 @@
+# Azure Virtual Desktop with Stack HCI
+
+Use the Full Configuration ARM templates if you are looking for more configurations in your Azure Virtual Desktop session hosts deployed on Azure Stack HCI.
+
+## Deploy  Azure Virtual Desktop on Azure Stack HCI
+This ARM template will setup Azure Virtual Desktop on Azure Stack HCI by creating a new host pool and workspace, creating the session hosts on the HCI cluster, joining the domain to downloading and installing the Azure Virtual Desktop agents and register them to the host pool.
+Check out the template [here](https://raw.githubusercontent.com/Azure/RDS-Templates/master/ARM-wvd-templates/HCI/QuickDeploy/CreateHciHostpoolQuickDeployTemplate.json)
+<a  href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fgithub.com%2FAzure%2FRDS-Templates%2Fblob%2Fmaster%2FARM-wvd-templates%2FHCI%2FQuickDeploy%2FCreateHciHostpoolQuickDeployTemplate.json"  target="_blank">
```

Comment:
Can this deploy button come in the next link, alongside with Visualize button?
This link is not redirecting, I believe we need to give the raw github link here.